app/presenters/single_content_item_presenter.rb

```diff
@@ -145,8 +145,8 @@ def useful_yes_no_total
   end
 
   def on_page_search_rate
-    metric_value_as_float = self.total_searches.delete(',').to_f
-    secondary_metric_value_as_float = self.total_pviews.delete(',').to_f
+    metric_value_as_float = @metrics['searches'][:value].to_f
+    secondary_metric_value_as_float = @metrics['pviews'][:value].to_f
```

Comment:
Can you get away with not calling `to_f` here? I'm wondering if it might be a number already.

Reply:
I'd also be tempted to change the variable names here as well, maybe you could use `searches`, or `searches_value` rather than `metric_value_as_float`, as I think that might make the following code easier to read?

Reply:
@cbaines it is an integer so we need to use to_f to avoid doing integer division. For example `9/5` returns `1` as opposed to `9.0/5.0` returning `1.8`.

Have changed the variable names, reads much better, thanks!

Reply:
Ah, that makes sense :+1: